### DISCUSSION TITLE
[Bug report] Post prefix is inserted twice for meta property="og:url"

### DISCUSSION THREAD

**Discussion by Enichan** [GetPublii/Publii#1820] [Bug Report] [answered]
### Operating system

Windows 10

### Publii version

0.46.2 (build 16968)

### Issue type

Application

### Bug description

When a posts prefix is set, the meta property "og:url" inserts the prefix twice, leading to the wrong url. For example on my blog posts have a "posts" prefix, and the og:url is this:

`<meta property="og:url" content="https://enikofox.com/posts/posts/introduction-to-bismuth-vm/" />`

Meanwhile the canonical link is correct:

`<link rel="canonical" href="https://enikofox.com/posts/introduction-to-bismuth-vm/">`

This is using the bundled simple theme.

EDIT: According to this issue https://github.com/GetPublii/Publii/issues/1767 this should have been fixed in 0.46.2 but I'm on that version and it's still happening.

### Steps to reproduce

- Install publii and use bundled simple theme
- Set a posts prefix under `Site settings` > `URLs`
- Create a post
- Preview changes, open the post, and view the source, the og:url property will have the prefix inserted twice

**Comment by dziudek** [comment, marked as answer]
@Enichan - this bug will be fixed in v.0.46.3 which will be released at the next week
